### DISCUSSION TITLE
Little Cover Pictures

### DISCUSSION THREAD

**Discussion by hjfisch** [rern/rAudio#1729] [Feature Requests]
I' missing the little Cover Pictures for quite a long Update-Versions
![Screenshot 2024-08-06 192738](https://github.com/user-attachments/assets/44e767b7-26b2-4a44-ac77-ae4fd74210c1)

**Comment by Sagnasty** [comment]
I also have the same issue for some time now.

**Comment by ccaper** [comment]
Same, I would love to get this feature back.
